### DISCUSSION TITLE
Installation Errors

### DISCUSSION THREAD

**Discussion by ficoquery** [cupcakearmy/cryptgeon#28] [Q&A] [answered]
I am currently having errors and cant source the cause.

Rust 1.60.0 Stable
Pnpm 6.32.6 
Docker Engine v20.10.14, Compose v2.4.1, containerd v1.5.11

Steps followed: https://github.com/cupcakearmy/cryptgeon/blob/main/README.md#development
_______

There are multiple errors I am getting after troubleshooting.

Error 1:

```
> error: no such subcommand: `watch`
> 
>         Did you mean `fetch`?
>  ELIFECYCLE  Command failed with exit code 101.
> ERROR: "dev:backend" exited with 1.
```

Error 2:

```

> sh: svelte-kit: command not found
>  ELIFECYCLE  Command failed.
>  WARN   Local package.json exists, but node_modules missing, did you mean to install?
>  ELIFECYCLE  Command failed with exit code 1.
> ERROR: "dev:backend" exited with 1.

```

Error 3:

`
> pnpm version not compatible with package.json.

`

**Comment by cupcakearmy** [comment, marked as answer]
Further discussed in #29
